### PR TITLE
MINOR: Update JavaDoc to use new API

### PR DESCRIPTION
**Pull request by bbejeck**

### CODE REVIEW

**Comment by bbejeck** [comment]
ping @mjsax @guozhangwang @dguy

File: streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java [hunk 1]
```diff
@@ -113,10 +113,10 @@
  * props.put(StreamsConfig.DEFAULT_VALUE_SERDE_CLASS_CONFIG, Serdes.String().getClass());
  * StreamsConfig config = new StreamsConfig(props);
  *
- * KStreamBuilder builder = new KStreamBuilder();
- * builder.stream("my-input-topic").mapValues(value -> value.length().toString()).to("my-output-topic");
+ * StreamsBuilder streamsBuilder = new StreamsBuilder();
```
Comment (mjsax) [inline streams/src/main/java/org/apache/kafka/streams/KafkaStreams.java]: Nit: `streamsBuilder` -> `builder`

**Comment by bbejeck** [comment]
Updated this

**Comment by guozhangwang** [comment]
retest this please

**Review by mjsax — APPROVED** [review]
LGMT.

@guozhangwang Can you merge this to `1.0` before cutting the RC1 ?

**Comment by guozhangwang** [comment]
LGTM. Merged to trunk and 1.0.

**Pull request closed without merging**